test/resources/must-not-be-null-doc-definitions.js

```diff
@@ -0,0 +1,145 @@
+function() {
```

Comment:
As noted in the pull request description, this file borrows from `required-doc-definitions.js`.